app/javascript/components/search/results/StudySearchResult.js

```diff
@@ -128,31 +129,50 @@ function facetMatchBadges(study) {
   </>)
 }
 
-/* displays a brief summary of a study, with a link to the study page */
-export default function SCPStudy({ study }) {
-  const termMatches = study.term_matches
-  const studyTitle= highlightText(study.name, termMatches).styledText
-  const studyDescription = formatDescription(study.description, termMatches)
-  const displayStudyTitle = { __html: studyTitle }
+// generate a cell count badge for SCP studies
+function cellCountBadge(study) {
+  if (study.study_source === 'SCP') {
+    return <span className="badge cell-count">
+      {study.cell_count} Cells
+    </span>
+  }
+}
 
-  let inferredBadge = <></>
-  if (study.inferred_match) {
+// generate the inferredBadge for SCP studies
+function inferredBadge(study, termMatches) {
+  if (study.inferred_match && study.study_source === 'SCP') {
     const helpText = `${termMatches.join(', ')} was not found in study metadata,
      only in study title or description`
-    inferredBadge = <span className="badge soft-badge" data-toggle="tooltip" title={helpText}>text match only</span>
+    return <span className="badge soft-badge" data-toggle="tooltip" title={helpText}>text match only</span>
   }
+}
+
+/* generate a badge to indicate to users the study origin for non-SCP studies */
+function studyTypeBadge(study) {
+  if (study.study_source === 'TDR') {
+    return <span className="badge badge-secondary study-type" data-toggle="tooltip"
+      title={'Study comes from Terra Data Repo'}> TDR </span>
+  }
+}
+
+/* displays a brief summary of a study, with a link to the study page */
+export default function StudySearchResult({ study }) {
+  const termMatches = study.term_matches
+  const studyTitle = highlightText(study.name, termMatches).styledText
+  const studyDescription = formatDescription(study.description, termMatches)
+  const displayStudyTitle = { __html: studyTitle }
 
   return (
     <>
       <div key={study.accession}>
-        <label htmlFor={study.name} id= 'result-title'>
-          <a href={study.study_url} dangerouslySetInnerHTML = {displayStudyTitle}></a>{inferredBadge}
+        <label htmlFor={study.name} id="result-title" className={'study-label'}>
+          <a href={study.study_url} dangerouslySetInnerHTML={displayStudyTitle} />
```

Comment:
Blocker: clicking on this link for TDR results does nothing -- it brings the user to the top of the page.  That strikes me as a UX bug.

Unless there are plans/designs on where TDR result links would go, let's not wrap their titles in a link.  This avoids confusing users who click on links and go nowhere.  Getting Jerome's input on this might be worthwhile!

Reply:
@eweitz unfortunately there is an issue on the TDR UI side where external hyperlinks do not resolve to an entity in the TDR UI.  Even if we had a valid link (which is unclear how we would generate it from the search results, as there is no reference to the snapshot/dataset), clicking it in a browser just dumps you on the home page, even after you've authenticated.  The TDR team took a look and they are apparently not caching the link origin after they complete the auth challenge.

Reply:
Thanks for the context!  So, indeed, we should not show TDR titles as links.